aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -51,7 +51,7 @@ Resources:
           resource_id: "AMICreate#{ami}",
           node_name: 'ami-${INSTANCE_ID}',
           run_list: [
-            local_mode ? 'recipe[cdo-apps]' : 'role[unmonitored-frontend]'
+            local_mode ? 'recipe[cdo-apps]' : 'role[front-end]'
```

Comment:
Previously, we didn't want the AMI instance itself to be sending monitoring-agent data to NewRelic during creation, since that instance isn't included in the set of servers actually serving live traffic. However, since #11868 we no longer do a full `chef-client` run on frontends as they're booting up, so we need to have the service enabled there.